contrib/runners/action_chain_runner/setup.py

```diff
@@ -23,6 +23,8 @@
 from dist_utils import fetch_requirements
 from dist_utils import apply_vagrant_workaround
 
+from action_chain_runner import __version__
+
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 REQUIREMENTS_FILE = os.path.join(BASE_DIR, 'requirements.txt')
```

Comment:
Since import doesn't work, can the version be read from the `../../../../../st2common/__init__.py` via similar "file" approach?
How worse or more hacky that could be, comparing to hardcoding same version number in all the runners?

Reply:
This would kinda break the whole package approach - it would rely on st2common to be in a specific path and always available with a runner package.

And it's not really hard coding - it's being updated via release workflows. It's exactly the same approach we follow for st2common and st2client. 

And this also allows us to separately version and release runner packages in the future (if we ever need to do that).

Reply:
I understand the purpose of st2common and st2client being versioned separately, but don't get the reason (yet) for runners which are part of the core and are in st2 repo code.

Any future plans to do extract these runners from the core and distribute separately?

Reply:
No plans for that right, but with package approach we do get more flexibility which allows us to do things like that in the future.

I believe @bigmstone did talk something about moving runners to standalone repos, but I personally don't want to do that any time soon yet (too much overhead).

Reply:
And also, with those changes they are not really versioned separately / interdependently - as mentioned above, it follows the same approach as other components and we bump / set the version during the release process.

Reply:
I just feel it's a bit of mess to change the version in 10 files with the same number without having plans to extract these parts from the core repo.
If there are no other options to brainstorm, - let it be :+1: 

Would just like @bigmstone or @m4dcoder to jump in and review the PR, as someone more familiar with st2 core and plans in context of runners.